scripts/bootstrap-db.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+#set -x
+
+# argument defaults
+DEPLOYMENT_ENV="dev"
+OXA_TOOLS_VERSION_OVERRIDE="master"
```

Comment:
nit: definitely a separate change, but I think we should remove all override logic. it's cumbersome